src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.ts

```diff
@@ -653,3 +667,19 @@ function cleanPastedText(text: string) : string {
 function normalize(text: string) : string {
     return text.normalize().replace(CrlfRe, "\n");
 }
+
+function focusAndOpenKeyboard(el: HTMLDivElement, timeout: number) {
+    const tempElement = document.createElement('input');
```

Comment:
afair there is editable div in our markup editor, so it should focusable by default, shouldn't it?

Reply:
it's a hack for iOS